examples/cloud-router-2-aws-connection/README.md

```diff
@@ -35,13 +24,8 @@ terraform init
 terraform apply
 ```
 
-To use this example of the module in your own terraform configuration include the following:
-
-*NOTE: terraform.tfvars must be a separate file, but all other content can be placed together in main.tf if you prefer*
-
-terraform.tfvars (Replace these values with your own):
+#terraform.tfvars.example
```

Comment:
Modify template to have the wording that was existing before the automation.

Reply:
And don't use the # because that's a reserved character in Readme's to denote Primary Headers.